WordPressAuthenticator/Sources/Signin/LoginNavigationController.swift

```diff
@@ -21,3 +20,25 @@ public class LoginNavigationController: RotationAwareNavigationViewController {
     }
 
 }
+
+// MARK: - RotationAwareNavigationViewController
+//
+public class RotationAwareNavigationViewController: UINavigationController {
```

Comment:
I moved it from `WordPressUI` to make sure `WordPressAuthentificator` doesn't re-exposes `WordPressUI` in its auto-generated header when creating an Objective-C module. I made some changes to remove its uses from Objective-C, that I kept, but then realized that there were a ton more usages from its test target, so I had to fix it.

Evidently, you can't include a Swift module in headers for an Objective-C module.